src/fosslight_binary/_binary_dao.py

```diff
@@ -19,6 +20,24 @@
 logger = logging.getLogger(constant.LOGGER_NAME)
 
 
+def move_owasp_info_to_comment_column(binary_item, row, bin_info_list):
```

Comment:
Let's remove bin_info_list

Reply:
Okay, but I changed the code and removed the function.
Check and review plz.